GDEdit Web/GDE.Web/Pages/GDAPI-Tests.razor

```diff
@@ -15,21 +15,23 @@
 
-    public string LevelTitle;
-    public string LevelDescription;
-    public string LevelObjects;
+    string LevelTitle;
```

Comment:
Class fields are private by default, but our convention is to explicitly state the accessibility.

Reply:
Theres really no reason to define the accessibility of the class in here as there is no other classes in this file.

Reply:
Never is there a reason to explicitly mark class/struct fields as private, since that's the default accessibility of the members. Same goes for functions; they are public by default. But we do explicitly state accessibility because that's the convention of the project, so we have to stick with it for consistency.